reducers/views/modals.js

```diff
@@ -26,6 +26,18 @@ function modalState(state = {}, action) {
             },
         };
 
+    case ActionTypes.TRACK_ANNOUNCEMENT_BAR:
```

Comment:
I think these should probably be in their own reducers since I don't think of the announcement bar as a modal.